cli/waiter/instance_select.py

```diff
@@ -0,0 +1,122 @@
+from enum import Enum
```

Comment:
The change in this file appear to be a move + file changes, any chance we can do those changes separately?  Will make reviewing the changes much easier

Reply:
The PR moves some common functionality from ssh.py to this new file o make them reusable and as a result, includes some changes to the ssh function calls.
Can you point them out? it will make it easier for me to fix them.

Reply:
Moved the import change to a different PR: https://github.com/twosigma/waiter/pull/1697